integration/travis/run_integration.sh

```diff
@@ -1,10 +1,24 @@
 #!/bin/bash
+
+# Relies on the COOK_EXECUTOR environment variable to choose which edn file to load.
+# When set to 1, the Cook executor is enabled.
+# Else, the Mesos Command executor is used.
+
 set -ev
 
 NOSE_ATTRIBUTES=${1:-'!explicit'}
 
 export PROJECT_DIR=`pwd`
 
+CONFIG_FILE="scheduler_config_command_executor.edn"
+if [ "${COOK_EXECUTOR}" = "1" ]
+then
+  CONFIG_FILE="scheduler_config_cook_executor.edn"
+fi
```

Comment:
Would having the `:executor` entry with `:command nil` have the same effect as not having the `:executor` entry at all? If so, I think we could get away with only one file and using `#config/env`